src/Parsing/Impl/Parser.cs

```diff
@@ -2120,6 +2145,19 @@ struct Name {
                         ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.DuplicateArgsDoubleArgumentErrorMsg);//duplicate ** args arguments
                     }
                     seenDictArg = true;
+                } else if (p.Kind == ParameterKind.PositionalOnlyMarker) {
+                    if (seenPositional) {
```

Comment:
Do we want to show all errors, i.e. if it has several issues like annotation, default and something else?

Reply:
I think the convention in the parser is to try and only show one error at a time per-character (if that makes sense), such that any given place will only be squiggled the one time.

Reply:
VS squiggles multiple so user can fix all errors rather than fix one, then see another. Technically squiggles should be different, i.e. annotation under `:`, default under `=`. Up to you.

Reply:
I'd prefer to stick with this for now, and come back to improve the errors overall if they overlap (since we have a number of cases like this already).